kraken-std/src/kraken/std/python/tasks/pytest_task.py

```diff
@@ -36,6 +43,7 @@ class PytestTask(EnvironmentAwareDispatchTask):
     doctest_modules: Property[bool] = Property.default(True)
     marker: Property[str]
     coverage: Property[CoverageFormat]
+    log_level: Property[LogLevel] = Property.default(LogLevel.INFO)
```

Comment:
You can use [the levels from the standard library](https://docs.python.org/3/library/logging.html#logging-levels), and then type this as:

```suggestion
    log_level: Property[Literal[logging.DEBUG, logging.INFO, ...]] = Property.default(logging.INFO)
```